.github/workflows/continuous-integration.yml

```diff
@@ -34,8 +34,6 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
-      - name: Update NPM
-        run: npm install -g npm
```

Comment:
This causes the same issue as https://github.com/loopbackio/loopback-connector-postgresql/pull/499#discussion_r853157977.

Reply:
Solved